spec/common/providers/https.spec.ts

```diff
@@ -133,6 +143,56 @@ async function runTest(test: CallTest): Promise<any> {
   expect(responseV2.status).to.equal(test.expectedHttpResponse.status);
 }
 
+function checkAuthContext(context: CallableContext, projectId: string) {
+  expect(context.auth).to.not.be.undefined;
+  expect(context.auth).to.not.be.null;
+  expect(context.auth.uid).to.equal(mocks.user_id);
+  expect(context.auth.token.uid).to.equal(mocks.user_id);
+  expect(context.auth.token.sub).to.equal(mocks.user_id);
+  expect(context.auth.token.aud).to.equal(projectId);
+  expect(context.instanceIdToken).to.be.undefined;
+}
+
+function checkAppCheckContext(
+  context: CallableContext,
+  projectId: string,
+  appId: string
+) {
+  expect(context.app).to.not.be.undefined;
+  expect(context.app).to.not.be.null;
+  expect(context.app.appId).to.equal(appId);
+  expect(context.app.token.app_id).to.be.equal(appId);
+  expect(context.app.token.sub).to.be.equal(appId);
+  expect(context.app.token.aud).to.be.deep.equal([`projects/${projectId}`]);
+  expect(context.auth).to.be.undefined;
+  expect(context.instanceIdToken).to.be.undefined;
+}
+
+function checkAuthRequest(request: CallableRequest, projectId: string) {
```

Comment:
nit: personally I feel a bit cleaner if we pass the mock user as a parameter rather than checkAuthRequest hard coding that it checks against a particular user. Not something I'd make a stink over though since we use the test fixture pretty regularly though.

Reply:
That's a fair point. Updated.